src/ripple/protocol/SystemParameters.h

```diff
@@ -38,22 +42,22 @@ systemName ()
 
 /** Configure the native currency. */
 static
-std::uint64_t const
+std::uint64_t constexpr
 SYSTEM_CURRENCY_GIFT = 1000;
 
 static
-std::uint64_t const
+std::uint64_t constexpr
 SYSTEM_CURRENCY_USERS = 100000000;
 
 /** Number of drops per 1 XRP */
 static
-std::uint64_t const
-SYSTEM_CURRENCY_PARTS = 1000000;
+tagged_integer<std::uint32_t, DropsTag> constexpr
```

Comment:
nit: previous type was `uint64_t` might as well keep it the same.

Reply:
Nice catch!

Reply:
FWIW, now that I'm rewriting, I've rediscovered why I changed it from 64- to 32-bit. There are several places where it's used as a multiplication factor for making a 32-bit value. That's perfectly fine when working with bare numbers, but these classes make it a little more complicated. It was easier to change here, since the value fit, than to do casts all over the place.